ngmix/bootstrap.py

```diff
@@ -6,2930 +6,164 @@
     - make it possible to specify the guess type (not just psf)
 
 """
-from pprint import pprint
```

Comment:
We should add all of the removed classes here to the changelog if we have not.

Reply:
I removed all of them, good enough to say "removed all the old bootstrapper and runner classes; replaced with generic Bootstrapper, Runner, PSFRuner, and  MetacalBootstrapper`, and associated functions?

Reply:
SGTM!